tests/test_amqp.py

```diff
@@ -11,12 +11,30 @@
 from openqabot import QEM_DASHBOARD
 from openqabot.amqp import AMQP
 
-namespace = namedtuple("Namespace", ["dry", "token", "openqa_instance", "url", "gitea_token"])
-args = namespace(True, "ToKeN", urlparse("http://instance.qa"), None, None)
+
+class Namespace(NamedTuple):
+    dry: bool
+    token: str
+    openqa_instance: str
+    url: str
+    gitea_token: str
+
+
+args = Namespace(
+    dry=True,
+    token="ToKeN",  # noqa: S106
+    openqa_instance=urlparse("http://instance.qa"),
+    url=None,
+    gitea_token=None,
+)
 amqp = AMQP(args)
 
-fake_method = namedtuple("Method", ["routing_key"])
-fake_job_done = fake_method("suse.openqa.job.done")
+
+class Method(NamedTuple):
```

Comment:
Maybe the name `Method` is a bit generic.


```suggestion
class FakeMethod(NamedTuple):
```

Reply:
Done